Podfile.lock

```diff
@@ -1,36 +1,20 @@
 PODS:
-  - OnTheRoad (0.9.0)
-  - Pelias (1.0.0-beta4):
-    - Pelias/Core (= 1.0.0-beta4)
-  - Pelias/Core (1.0.0-beta4)
+  - OnTheRoad (1.0.0)
+  - Pelias (1.0.0):
+    - Pelias/Core (= 1.0.0)
+  - Pelias/Core (1.0.0)
```

Comment:
Pelias defines a "Core" subspec which OnTheRoad and Tangram-es do not. Your app's Podfile can define different versions for each subspec (although normally you try to avoid such things).